sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,207 @@ def __repr__(self):
+        self.mu_k = mu_k if mu_k is not None else 1
+        self.mu_s = mu_s if mu_s is not None else self.mu_k
+        self.f_n = f_n
+        self.sigma = sigma if sigma is not None else 0
+        self.v_s = v_s if v_s is not None or v_s == 0 else 0.01
+        self.pathway = pathway
+        self.s_t = s_t if s_t != 100 else 100
+
+    @property
+    def mu_k(self):
+        return self._mu_k
+
+    @mu_k.setter
+    def mu_k(self, mu_k):
+        if hasattr(self, '_mu_k'):
+            msg = (
+                f'Can\'t set attribute `mu_k` to '
+                f'{repr(mu_k)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._mu_k = sympify(mu_k, strict=True)
+
+    @property
+    def mu_s(self):
+        return self._mu_s
+
+    @mu_s.setter
+    def mu_s(self, mu_s):
+        if hasattr(self, '_mu_s'):
+            msg = (
+                f'Can\'t set attribute `mu_s` to '
+                f'{repr(mu_s)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._mu_s = sympify(mu_s, strict=True)
+
+    @property
+    def f_n(self):
+        return self._f_n
+
+    @f_n.setter
+    def f_n(self, f_n):
+        if hasattr(self, '_f_n'):
+            msg = (
+                f'Can\'t set attribute `f_n` to '
+                f'{repr(f_n)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._f_n = sympify(f_n, strict=True)
+
+    @property
+    def sigma(self):
+        return self._sigma
+
+    @sigma.setter
+    def sigma(self, sigma):
+        if hasattr(self, '_sigma'):
+            msg = (
+                f'Can\'t set attribute `sigma` to '
+                f'{repr(sigma)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._sigma = sympify(sigma, strict=True)
+
+    @property
+    def v_s(self):
+        return self._v_s
+
+    @v_s.setter
+    def v_s(self, v_s):
+        if hasattr(self, '_v_s'):
+            msg = (
+                f'Can\'t set attribute `v_s` to '
+                f'{repr(v_s)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._v_s = sympify(v_s, strict=True)
+
+    @property
+    def s_t(self):
+        return self._s_t
+
+    @s_t.setter
+    def s_t(self, s_t):
+        if hasattr(self, '_s_t'):
+            msg = (
+                f'Can\'t set attribute `s_t` to '
+                f'{repr(s_t)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._s_t = sympify(s_t, strict=True)
+
+    @property
+    def force(self):
+        v = self.pathway.extension_velocity
+        f_c = self.mu_k * self.f_n
+        f_max = self.mu_s * self.f_n
+        stribeck_term = (f_max - f_c) * exp(-(v / self.v_s)**2) if self.v_s != 0 else 0
+        viscous_term = self.sigma * v * -tanh(self.s_t * v) if self.f_n != 0 else 0
+        return f_c * -tanh(self.s_t * v) + stribeck_term + viscous_term
```

Comment:
Though `tanh` may give nicer results when integrating by example, it is mathematically incorrect. If a user would like to smoothen the result, then it is better to let him/her replace the `sign` in the final equations manually.